core-addon/Core.js

```diff
@@ -42,9 +42,20 @@ module.exports = class Core {
     this._connectionPort = null;
   }
 
+  async setUninstallURL() {
+    // set the URL to redirect when a user uninstalls Rally.
+    const deletionId = await this._storage.getDeletionID();
```

Comment:
Why do we expect the ID to be unavailable? Can you document our expectations here?

Reply:
I'll add a comment, but it's first set during enrollment so if you have the core add-on installed but have not enrolled then there is neither enrollment nor deletion pings set, yet. In this case we still want to show the offboarding URL on add-on uninstall, but there's no data to delete (please correct if I am misunderstanding though)

Reply:
Makes sense :) Let's just leave a comment so that we don't forget in N months ;)